assets/_globalCss.css

```diff
@@ -1,49 +1,84 @@
-@import 'normalize.css/normalize.css';
+@import 'tailwindcss/base';
+@import 'tailwindcss/components';
+@import 'tailwindcss/utilities';
+
 @import '../public/assets/fonts/inter/inter.css';
 
+@layer base {
+    :root {
+        --background: 0 0% 100%;
+        --foreground: 0 0% 0%;
+        --card: 0 0% 100%;
+        --card-foreground: 0 0% 0%;
+        --popover: 0 0% 100%;
+        --popover-foreground: 0 0% 0%;
+        --primary: 0 0% 0%;
+        --primary-foreground: 0 0% 100%;
+        --secondary: 0 0% 20%;
+        --secondary-foreground: 0 0% 80%;
+        --muted: 0 0% 40%;
+        --muted-foreground: 0 0% 60%;
+        --accent: 264 100% 50%;
+        --accent-foreground: 244 100% 70%;
+        --destructive: 348 78% 50%;
+        --destructive-foreground: 348 98% 70%;
+        --border: 0 0% 90%;
+        --input: 0 0% 90%;
+        --ring: 0 0% 0%;
+        --radius: 0.25rem;
+    }
+}
+
 :root {
-    /* color definitions */
-    --color-background: rgb(255 255 255);
-    --color-foreground: rgb(0 0 0);
-    --color-disabled: rgb(128 128 128);
-    --color-selection: rgb(255 240 170);
-    --color-notification: rgb(255 96 96);
+    /* color definitions (legacy support) */
+    --color-background: hsl(var(--background));
+    --color-foreground: hsl(var(--foreground));
+    --color-disabled: hsl(var(--muted));
+    --color-notification: hsl(var(--accent));
     --color-background-alpha: hsl(0deg 0% 97%);
     --color-background-beta: hsl(0deg 0% 94%);
     --color-foreground-alpha: rgb(0 0 0 / 5%);
 
-    /* size definitions */
-    --border-radius: 4px;
+    /* size definitions (legacy support) */
+    --border-radius: var(--radius);
     --margin: 1rem;
     --line-height: 1.3;
     --icon-size: calc(1rem * var(--line-height));
 }
 
 @media (prefers-color-scheme: dark) {
     :root {
-        --color-background: hsl(0deg 0% 8%);
-        --color-foreground: rgb(255 255 255);
-        --color-disabled: rgb(128 128 128);
-        --color-selection: rgb(128 196 255);
-        --color-notification: rgb(96 64 255);
+        --background: 0 0% 0%;
```

Comment:
We overlooked something here and accidentally changed `8%` to `0%`. Fixed in https://github.com/medienhaus/medienhaus-spaces/commit/f5bb18d028c6399e00547cdc1f64a421079bd050.